src/Protocol.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef SONAR_OCULUS_M750D_PROTOCOL_HPP
+#define SONAR_OCULUS_M750D_PROTOCOL_HPP
+
+#include "Oculus.h"
```

Comment:
You cannot have this included in a `.hpp`. It's not namespaced and will pollute every client down the way.

Include it only in the `.cpp`